spec/requests/partners_spec.rb

```diff
@@ -0,0 +1,135 @@
+require "rails_helper"
+
+RSpec.describe "Partners", type: :request, include_shared: true do
+  let(:partner) { create(:partner) }
```

Comment:
My personal preference is to avoid `let` block for reasons that are better articulated in [this blog post](https://thoughtbot.com/blog/lets-not) than I could do in github comments.